reposerver/repository/repository_test.go

```diff
@@ -2663,17 +2663,28 @@ func runGit(t *testing.T, workDir string, args ...string) string {
 	return stringOut
 }
 
-func Test_findHelmValueFilesInPath(t *testing.T) {
+func Test_walkHelmValueFilesInPath(t *testing.T) {
```

Comment:
Should there be a new test case to confirm that values files are detected in directories under the root directory?

Reply:
ohh good idea. I just added an additional dir and bumped the assert to 5.